hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/ErrorMessage.java

```diff
@@ -12,7 +12,7 @@
 	HL7Error_Msg_EncryptionError("TXFR029E", "Encryption protocols failed with remote facility."),
 	
 	HL7Error_Msg_InvalidMSHSegment("HNPS002E", "Invalid MSH segment format"),
-	HL7Error_Msg_UnknownReceivingApplication("HNPS005E", "Unknown receiving application"),
+	HL7Error_Msg_UnknownReceivingApplication("HNPS005E", "Unknown receiving application:"),
```

Comment:
Is the text for HL7Error_Msg_FacilityIDMismatch supposed to be reformatted as well? It currently ends with a period.